mobile/lib/main.dart

```diff
@@ -163,6 +169,7 @@ Future<void> _runMinimally(String taskId, TimeLogger tlog) async {
   LocalFileUpdateService.instance.init(prefs);
   await LocalSyncService.instance.init(prefs);
   RemoteSyncService.instance.init(prefs);
+  await FavoritesService.instance.initFav();
```

Comment:
This is expensive call. Is there reason for adding this? We can also use FavoritesServiceInitCompleteEvent event if needed.